src/types.ts

```diff
@@ -30,28 +30,41 @@ export type BabelConfig = _babel.TransformOptions
 export interface TsJestGlobalOptions {
   /**
    * Compiler options. It can be:
-   *     - `true` (or `undefined`, it's the default): use default tsconfig file
-   *     - `false`: do NOT use default config file
-   *     - `path/to/tsconfig.json`: path to a specific tsconfig file (<rootDir> can be used)
-   *     - `{...}`: an object with inline compiler options
+   * - `false`: do NOT use default config file
```

Comment:
These bullets were not showing up correctly in Intellisense. This fixes that.

<img width="701" alt="Screen Shot 2020-04-27 at 4 34 45 PM" src="https://user-images.githubusercontent.com/1058243/80423116-03071a80-88a5-11ea-88c0-975f485da198.png">